### PR TITLE
Fix/clean test names

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
Remove not needed name parts from UI-Tests.

### CODE REVIEW

**Review by marcussc — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan [hunk 1]
```diff
@@ -106,7 +106,31 @@
         "ENAUITests_07_ContactJournal\/testOverviewScenario3()",
         "ENAUITests_07_ContactJournal\/testOverviewScenario4()",
         "ENAUITests_07_ContactJournal\/test_AddTwoPersonsAndOneLocationToDate()",
-        "ENAUITests_07_ContactJournal\/test_Overview()"
+        "ENAUITests_07_ContactJournal\/test_Overview()",
+        "ENAUITests_ContactJournal\/testAddLocationToDate()",
```
[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan:110]: I'm also wondering about that. The old names are still in the file but don't show up in the UI.

[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan:110]: I think I need to remove them manually from the file content.

[the remark this answers is not in the document]
Comment (naveeddotio) [reply src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan:91]: It's a bug in Xcode.

File: src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan [hunk 2]
```diff
@@ -106,7 +106,31 @@
...
+        "ENAUITests_ContactJournal\/testNavigationToInformationVC()",
+        "ENAUITests_ContactJournal\/testOpenExportViaSheet()",
+        "ENAUITests_ContactJournal\/testOpenInformationScreenViaSheet()",
+        "ENAUITests_ContactJournal\/testOverviewScenario1()",
+        "ENAUITests_ContactJournal\/testOverviewScenario2()",
+        "ENAUITests_ContactJournal\/testOverviewScenario3()",
+        "ENAUITests_ContactJournal\/testOverviewScenario4()",
+        "ENAUITests_ContactJournal\/test_AddTwoPersonsAndOneLocationToDate()",
+        "ENAUITests_ContactJournal\/test_Overview()",
+        "ENAUITests_ContactJournal\/test_screenshot_AddTwoPersonsTwoLocations()",
```
[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan]: Do you mean ENAUITests target?

[the remark this answers is not in the document]
Comment (marcussc) [reply src/xcode/ENA/TestPlans/ContactJournalUITests.xctestplan]: Screenshot 2022-03-17 at 10 11 47
Whatever this is called? Target? Testtarget?

**Pull request closed without merging**